src/interaction.wiki

```diff
@@ -1,6 +1,6 @@
 =Implementing Web Interaction Using Eliom=
 
-//The code of this tutorial has been tested with Eliom 4.// \\
+//The code of this tutorial has been tested with Eliom dev.// \\
```

Comment:
"dev" is a moving target. Whenever you say "dev", can you also provide a date (or commit hash)?